crates/trie/src/state.rs

```diff
@@ -86,18 +84,24 @@ impl HashedPostState {
             }
         }
 
-        for (address, storage) in storages {
-            // The `wiped` flag indicates only whether previous storage entries should be looked
-            // up in db or not. For reverts it's a noop since all wiped changes had been written as
-            // storage reverts.
-            let hashed_storage = HashedStorage::from_iter(
-                false,
-                storage.into_iter().map(|(slot, value)| (keccak256(slot), value)),
-            );
-            this.storages.insert(keccak256(address), hashed_storage);
-        }
+        let hashed_accounts = HashMap::from_iter(
+            accounts.into_iter().map(|(address, info)| (keccak256(address), info)),
```

Comment:
could this benefit from rayon?

Reply:
i'm unsure about exact size yet, so stayed clear of rayon for now